js/scripts/zone_loaders/village_zone_loaders.js

```diff
@@ -121,10 +129,10 @@ const loadVillageField = () => {
 
 
     }
-    STORY.huntingInvite = true;
-    if (STORY.huntingInvite) {
+    
+    if (STORY.invitedHunting) {
         GAME.addEntity(new PapaChad(
-            new Vector(ZONE.MAX_PT.x - 2 * PapaChad.SCALED_SIZE.x, ZONE.MAX_PT.y - 30 * Block.SCALED_SIZE),
```

Comment:
Was this the whole problem with his spawning?

Reply:
More the if statement combined with his falling speed